### PR TITLE
BC-6672 fix incorrect translation in alert

### PR DESCRIPTION
**Pull request by MichalNers**
# Description

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: locales/calendar/es.json
Comment (bn-pass) [inline locales/calendar/es.json:1]: What about files with german and english translations? I only see `es.json` and `uk.json` in `locales/calendar` dir.

**Review by bn-pass — CHANGES_REQUESTED** [review]

File: .prettierrc.js
[the remark this answers is not in the document]
Comment (MichalNers) [reply .prettierrc.js:1]: right !

File: helpers/momentHelper.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	w: 'a week',
+	ww: '%d weeks',
+	y: 'a year',
+};
+*/
+
+const createCustomRelativeTimeConfig = (localFile) => ({
+	d: `${localFile['moment.relativeTime.aDay']}`,
+	dd: `%d ${localFile['moment.relativeTime.days']}`,
+	future: `${localFile['moment.relativeTime.futureIn']} %s`,
```
[the remark this answers is not in the document]
Comment (MichalNers) [reply helpers/momentHelper.js:28]: image
looks good

File: locales/calendar/es.json
Comment (MichalNers) [reply locales/calendar/es.json:1]: Moment library  has translated English and German

**Review by bn-pass — APPROVED** [review]

**Review by CeEv — APPROVED** [review]

**Review by CeEv — APPROVED** [review]

**Pull request merged**